elpd_dashboard.ipynb

```diff
@@ -0,0 +1,811 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    y_max_fi = pn.widgets.FloatInput(name='y_max', value=0, step=0.1, start=-100, end=100, width=200)
</pre><p>Maybe we can just remove this controls and let matplotlib to automatically adjust the y.axis?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz_dashboard/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>